### DISCUSSION TITLE
Displaying Plotly on Page

### DISCUSSION THREAD

**Discussion by ddlidded** [requarks/wiki#2805] [Help / Questions]
I am attempting to use the basic code (JS) example provided by plotly but it is proving hard to get it to work after adding the code to the text editor in the code option. Am I missing something or does it not allow JS to load from inside?

**Comment by ghost** [comment]
You haven't provided the required info about your host! (OS, Wiki.js version, Database engine)

**Comment by ddlidded** [comment]
Docker built with Caprover

Using Postgres
OS: Ubuntu 18 But on Docker
Version is the latest 2.5.170
